pages/api/dashboard/index.ts

```diff
@@ -118,24 +119,29 @@ const getAllMonths = function (transactions: any[]): AllMonths {
 }
 
 const getUserDashboardData = async function (userId: string): Promise<DashboardData> {
-  const buttonsCount = (await paybuttonService.fetchPaybuttonArrayByUserId(userId)).length
+  const buttonsCount = (await paybuttonService.fetchPaybuttonArrayByUserId(userId))
```

Comment:
Maybe rename it to just `buttons` since is not the count anymore

Reply:
For sure, added